### DISCUSSION TITLE
Why does the loco generate model command place the newest model at the top of migration/src/lib.rs instead of at the bottom?

### DISCUSSION THREAD

**Discussion by hdoo42** [loco-rs/loco#1108] [Q&A]
My question is about why this command places the new file at the top of the migration sequence. 

When it’s placed at the top, it will be applied first during a migration and will be the last to be reverted during a down operation. In my opinion, the reverse order would make more sense. As far as I know, SeaORM typically adds new migrations at the bottom. I’m curious to know why loco does it this way.

**Comment by kaplanelad** [comment]
Thanks, @hdoo42! Could you please open an issue for this?
